package.json

```diff
@@ -34,10 +34,15 @@
   },
   "dependencies": {
     "@nestjs/common": "^8.0.0",
+    "@nestjs/config": "^2.1.0",
     "@nestjs/core": "^8.0.0",
+    "@nestjs/mapped-types": "^1.0.1",
     "@nestjs/platform-express": "^8.0.0",
+    "@nestjs/typeorm": "^9.0.0",
```

Comment:
you have core nest packages for v8 and installed typeorm v9. Chris upgraded his project to 9 so you can do the same here

Reply:
You can try using npm-check-updates to install any updates for Nest. The following should help you out: 

1. `npm install npm-check-updates`
2.  `npm-check-updates "/@nestjs*/" -u`
3.  `npm install`